providers/push-webhook/src/lib/push-webhook.provider.ts

```diff
@@ -15,13 +15,20 @@ export class PushWebhookPushProvider implements IPushProvider {
     private config: {
       hmacSecretKey?: string;
       webhookUrl: string;
-    }
+    },
   ) {}
 
   async sendMessage(
-    options: IPushOptions
+    options: IPushOptions,
   ): Promise<ISendMessageSuccessResponse> {
-    const bodyData = this.createBody(options);
+    const bodyData = this.createBody({
+      ...options,
+      payload: {
+        ...options.payload,
+        subscriber: options.subscriber,
+        step: options.step,
+      },
```

Comment:
for the `webhook` provider send `subscriber, step` as the payload, like we did before